Sources/Quick/Configuration/Configuration.swift

```diff
@@ -72,10 +72,16 @@ final public class Configuration: NSObject {
         provided with metadata on the example that the closure is being run
         prior to.
     */
+#if !os(Linux)
```

Comment:
Oh neat, I'll try to track down those macro names so they can be used instead. Agreed that just checking for Linux is imprecise and not very future-proof.